Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -366,14 +366,15 @@ extension Stripe.Plan {
   public static let individualYearly = mock
     |> \.amount .~ 170_00
     |> \.id .~ .individualYearly
+    |> \.interval .~ .year
```

Comment:
these mock plans weren't stubbing out the interval. think we need this right?

Reply:
Yeah :/ good catch!